tests/suites/target_test.function

```diff
@@ -384,8 +384,8 @@ int execute_tests( int args, const char ** argv )
     while ( 1 )
     {
         ret = 0;
-        test_info.result = TEST_RESULT_SUCCESS;
-        test_info.step = (unsigned long)( -1 );
+        mbedtls_test_info.result = MBEDTLS_TEST_RESULT_SUCCESS;
```

Comment:
Call to `mbedtls_test_info_reset()` here as well?

Reply:
Done.